### PR TITLE
beta-20160629 release notes

### PR DESCRIPTION
**Pull request by jseldess**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/429)

### CODE REVIEW

File: beta-20160629.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+toc: false
+summary: Additions and changes in CockroachDB version beta-20160630.
+---
+
+## Jun 29, 2016
+
+### New Features
+
+- A prototype implementation of `JOIN` (non-optimized) is now available. [#7202](https://github.com/cockroachdb/cockroach/pull/7202)
+- [Column Families](https://github.com/cockroachdb/cockroach/blob/master/docs/RFCS/sql_column_families.md) are a new, more efficient key-value representation of SQL tables. The [FAMILY](column-families.html) syntax extension allows users to tune column families, when necessary. [#7466](https://github.com/cockroachdb/cockroach/pull/7466), [#7408](https://github.com/cockroachdb/cockroach/pull/7408)
```
Comment (tschottdorf) [inline beta-20160629.md]: No data migration is necessary: the previous format corresponds to a separate column family for each column in the table.

**Comment by tschottdorf** [comment]
LGTM

File: _data/sidebar_doc.yml [hunk 2]
```diff
@@ -126,8 +129,5 @@ entries:
         - title: beta-20160602
           url: /beta-20160602.html
 
-        - title: beta-20160526
```
Comment (tamird) [inline _data/sidebar_doc.yml]: what's this about?

Comment (tamird) [inline _data/sidebar_doc.yml]: ah, it moved. disregard!

**Comment by jseldess** [comment]
Review status: 0 of 5 files reviewed at latest revision, 1 unresolved discussion.

---

*[beta-20160629.md, line 12 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/429#-KLYEZzCGwYP1eAuT4SD-r1-12:-KLYJFaRmb0gUHBmWlxe:-499146009) ([raw file](https://github.com/cockroachdb/docs/blob/cbbf41f6b1a6447242c03f9989d6cff0e3ad5acc/beta-20160629.md#L12)):*
<details><summary><i>Previously, tschottdorf (Tobias Schottdorf) wrote…</i></summary>
> No data migration is necessary: the previous format corresponds to a separate column family for each column in the table.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/429)*

**Pull request merged**